src/test/java/edu/ucsb/mapache/controllers/SlackSlashCommandControllerTests.java

```diff
@@ -267,11 +276,25 @@ public void test_googleSearchMultipleArguments() throws Exception {
         .andExpect(status().is(200));
   }
 
+  @Test
+  public void test_googleSearch_message_format_0() throws Exception {
```

Comment:
Rename test to be a better name since this is actually just testing the case of no results. Would like Bryan to do a review since I'm not as familiar with this